docs/appendices/release-notes/5.10.4.rst

```diff
@@ -44,6 +44,9 @@ See the :ref:`version_5.10.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed ram-accounting for upsert statements to prevent nodes from running
+  out of memory on large bulk updates.
```

Comment:
I think it would be nice to be a bit more detailed here. E.g.:
```suggestion
- Fixed ram-accounting accuracy for upsert statements by using the table stats
  to estimate the average row size. If stats are not (yet) available, a rough estimate
  based on the table columns data types is used. By this, nodes should be prevented
  from running out of memory on large bulk updates.
```

Reply:
👍